src/main/java/com/example/bookstore/mapper/BookMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.mapper;
+
+import com.example.bookstore.config.MapperConfiguration;
+import com.example.bookstore.dto.BookDto;
+import com.example.bookstore.dto.CreateBookRequestDto;
+import com.example.bookstore.model.Book;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfiguration.class)
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toModel(CreateBookRequestDto requestDto);
```

Comment:
I'd recommend you add `@Mapping(target = "id", ignore = true)` above this method to show, that the `id` has to be ignored